Makefile

```diff
@@ -1,13 +1,15 @@
 TEST_TAGS?=test
 CONTAINER_BUILD_OPTS ?= --build-arg=quay_expiration=2d
 CONTAINER_IMAGE ?= provisioning-backend
+PACKAGE_BASE = github.com/RHEnVision/provisioning-backend/internal
+LDFLAGS = "-X $(PACKAGE_BASE)/version.BuildCommit=$(shell git rev-parse --short HEAD) -X $(PACKAGE_BASE)/version.BuildTime=$(shell date +'%Y-%m-%d_%T')"
 
 .PHONY: build
 build: build-pbapi build-pbmigrate
 
 .PHONY: build-pbapi
 build-pbapi:
-	CGO_ENABLED=0 go build -o pbapi ./cmd/pbapi
+	CGO_ENABLED=0 go build -ldflags $(LDFLAGS) -o pbapi ./cmd/pbapi
 
```

Comment:
We do not have a target for pbworker, I will add it.

Reply:
should we add it in a separate PR, I believe we should also put that in the container, right?

Reply:
Well, I have no intentions working on the container right now. A target makes sense even if we don't have a container yet.

Reply:
Tho isn't this just about putting the binary into the container? That is like literally one line.

Reply:
Hmm there's more than that, the problem is entrypoint. I have no idea how to solve at the moment.

I am adding only Makefile target, as a separate commit in this PR if you don't mind.